homeassistant/components/technove/typing.py

```diff
@@ -0,0 +1,7 @@
+"""Typings for the TechnoVE integration."""
```

Comment:
You can just have this declaration in `__init__.py`. in the coordinator you would need to import the ConfigEntry in a `if TYPE_CHECKING:` block and you would need to add `from __future__ import annotations`

Reply:
Done.